pyro/poutine/handlers.py

```diff
@@ -39,11 +39,11 @@
     >>> traced_model = poutine.trace(conditioned_model)
 
 Many inference algorithms or algorithmic components can be implemented
-in just a few lines of code:
+in just a few lines of code::
 
-    >>> guide_tr = poutine.trace(guide).get_trace(...)
-    >>> model_tr = poutine.trace(poutine.replay(conditioned_model, trace=tr)).get_trace(...)
-    >>> monte_carlo_elbo = model_tr.log_prob_sum() - guide_tr.log_prob_sum()
+    guide_tr = poutine.trace(guide).get_trace(...)
```

Comment:
Putting this into a literal block without `>>>` so as to avoid running doctest on this snippet.